### PR TITLE
Adds "Pet care" request type

### PR DESCRIPTION
**Pull request by cheshireoctopus**
https://trello.com/c/ZPUbqBV8/10-add-pet-care-option-to-request-type

## Overview

This PR adds `Pet care` as a provider request type

### CODE REVIEW

File: app/javascript/packs/pages/provider_signup/ProviderRequestForm.jsx [hunk 1]
```diff
@@ -46,6 +46,10 @@ const providerRequestTypes = [
   {
     label: "Supplies",
     value: "supplies"
+  },
+  {
+    label: "Pet care",
+    value: "petCare"
   }
```
Comment (cheshireoctopus) [inline app/javascript/packs/pages/provider_signup/ProviderRequestForm.jsx:53]: Unsure of where in the checklist this item should live. Items are not sorted alphabetically.

Further, thoughts on a `value` of `petCare` vs `pets`?

Comment (cantino) [reply app/javascript/packs/pages/provider_signup/ProviderRequestForm.jsx:53]: No strong opinion, we could do `pets` for simplicity.

Comment (cheshireoctopus) [reply app/javascript/packs/pages/provider_signup/ProviderRequestForm.jsx:53]: Changed 👍

**Comment by cantino** [comment]
Thanks @cheshireoctopus!

**Pull request merged**